src/bundles/communication/Communications.ts

```diff
@@ -0,0 +1,123 @@
+import context from 'js-slang/context';
+import { MultiUserController } from './MultiUserController';
+import { GlobalStateController } from './GlobalStateController';
+import { RpcController } from './RpcController';
+
+class CommunicationModuleState {
+  multiUser: MultiUserController;
+  globalState: GlobalStateController | null = null;
+  rpc: RpcController | null = null;
+
+  constructor(address: string, port: number) {
+    let multiUser = new MultiUserController();
+    multiUser.setupController(address, port);
+    this.multiUser = multiUser;
+  }
+}
+
+let moduleState = new CommunicationModuleState('broker.hivemq.com', 8884);
```

Comment:
Can this be a `const` instead? Looking through the file, I don't see it being reassigned anywhere. Unless we plan to allow the user to specify which broker to use, in which case we should also allow users to specify the port number in addition to the host.